chapters/data-processing.tex

```diff
@@ -0,0 +1,784 @@
+%------------------------------------------------
+
+\begin{fullwidth}
+
+% Chapter overview
+The tasks described in this chapter are called by many different names.
+Some would say data cleaning, others data munging, or maybe data wrangling.
+It means transforming raw data into a more convenient format for your intended use.
+Although original data typically requires more extensive data cleaning than secondary data,
+it is broadly agreed that this is, on average, the most time-consuming step in a project's data work \citep{dasu2003exploratory}.
+This chapter describes the different tasks involved in the cleaning process,
+and outlines the workflow that our team adopts to make it efficient and transparent.
+
+% What is data cleaning
+Implementing the tasks described in this chapter will make data analysis much easier.
+The first section will guide on making your data \textit{tidy},
+so the relationship between rows and columns in a data set is well-defined.
+This includes removing duplicated observations.
+To assert the accuracy of your data, it must then go through quality assurance checks,
+as described in the second section.
+After that, identifying information is typically no longer needed,
+the following section will outline steps to remove direct identifiers.
+Finally, the last section discussed how to explore each variable in your dataset and 
+make sure that it is as well documented and easy to use as possible.
+
+
+\end{fullwidth}
+
+%------------------------------------------------
+
+\subsection{Before you get started}
+
+% Don't mess with the raw data
+The starting point for all tasks described in this chapter is the raw data,
+containing the exact files received, with no changes or additions.
+Be mindful of how and where these files are stored
+as they cannot be re-created and nearly always contain confidential information.
+Raw data files must \textit{never} be edited directly.
+If the raw data contains confidential information,
+any tasks that precede de-identification must be carried on encrypted data.
+ 
+% Why cleaning is necessary
+Raw data files may be received in a variety of formats, 
+and are often not easy to handle using statistical software.
+The data cleaning tasks described in this chapter will prepare the raw data
+for quantitative analysis.
+To make this complex process less error-prone,
+we recommend that no changes be made to the content of data at this point
+(with one exception that will be carefully discussed).
+Correcting outliers, imputing values and creating derived indicators
+are all research decisions, and are therefore considered part of analysis.
+Therefore, the main output of data cleaning, the cleaned dataset,
+contains almost exactly same information as the raw data,
+but in a format that is easier to interact with.  
+
+\section{Identifying the identifier(s)}
+
+% Intro
+The very first step in creating statistical software-friendly data sets
```

Comment:
maybe "analysis-ready" rather that "statistical software-friendly"? although in practice the latter is what we are doing in practice, it isn't exactly the objective

Reply:
I also got caught on "statistical software-friendly". I like "analysis-ready" better, but it is not ready for analysis as we have construction in-between. Would "analysis-friendly" be better?